metron-interface/metron-rest/README.md

```diff
@@ -353,6 +355,20 @@ Request and Response objects are JSON formatted.  The JSON schemas are available
       * searchRequest - Search request
   * Returns:
     * 200 - Search results
+    
```

Comment:
Such as complex types.

This comes through here, because we are creating our own Field Type Enums